### ISSUE TITLE
Build intermediate footer

### ISSUE THREAD

**Issue by simonwex** [adobe/developer.adobe.com#76]
Building on #72, let's build some stand-alone markup/css/js that can be used as a shared footer for both the helix and AEM versions of the site.

**Comment by filmaj** [comment]
And I believe the footer screenshot is the following:

![footershot](https://user-images.githubusercontent.com/25260/50235160-1aa6c880-036c-11e9-9d2b-66cf58168158.png)

**Comment by filmaj** [comment]
Found the mobile version.

I've pushed a preliminary, non-interactive mobile-only version to the `navs` branch:

<img width="422" alt="screen shot 2019-02-01 at 12 56 46 pm" src="https://user-images.githubusercontent.com/52645/52143363-dec87980-2620-11e9-8ac1-4ec3eb15bf49.png">

It's a little different from the mockups mostly because I'm trying not to veer away too much from default Spectrum styles. So that's what contributes to e.g. chevrons for expanding the items being on the left rather than the right. Not sure if we should have a convo on if we want to consolidate these kinds of differences, or if we clobber over Spectrum's guidelines and go ahead with the dev site mockups as-is.
